### PR TITLE
BMI Calculator UI Updated

### LINKED ISSUE #35: BMI calculator UI Update
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

I will make a BMI calculator where if you add your age, height and weight it will tell your BMI and whether you are healthy or not.
Please assign this issue to me @Rakesh9100 

### Screenshots

_No response_

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### PR DESCRIPTION
**Pull request by VardaPanchal005**
# Fixes Issue🛠️

Closes #35 

# Description👨‍💻 

Updated the UI and added Age feature

# Type of change📄

- [X] New feature (non-breaking change which adds functionality)

# How this has been tested✅

# Checklist✅ 

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have added demonstration in the form of GIF/video file
- [X] I am an Open Source Contributor

# Screenshots/GIF📷
Screenshot 2024-01-06 232410

### CODE REVIEW

File: Calculators/BMI-Calculator/BMI-Calculator.html [hunk 1]
```diff
@@ -2,32 +2,59 @@
...
 
 <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" type="text/css" href="BMI-Calculator.css" />
-    <title>BMI Calculator - CalcDiverse</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
```
Comment (Rakesh9100) [inline Calculators/BMI-Calculator/BMI-Calculator.html:7]: Pls change the file name here @VardaPanchal005

File: Calculators/BMI-Calculator/BMI-Calculator.html [hunk 2]
```diff
@@ -2,32 +2,59 @@
...
-
-    <script src="BMI-Calculator.js"></script>
+    </div>
+    <div id="myModal" class="modal">
+        <div class="modal-content">
+            <span class="close">&times;</span>
+            <p id="modalText"></p>
+        </div>
+    </div>
+    <script src="script.js"></script>
```
Comment (Rakesh9100) [inline Calculators/BMI-Calculator/BMI-Calculator.html:57]: Also change the file name here @VardaPanchal005

File: Calculators/BMI-Calculator/README.md [hunk 3]
```diff
@@ -12,4 +12,3 @@ Calculates the Body Mass Index of a person.
 
 ## Screenshots :-
```
Comment (Rakesh9100) [inline Calculators/BMI-Calculator/README.md:14]: Add the proper screenshot here @VardaPanchal005

File: Calculators/BMI-Calculator/BMI-Calculator.html (hunk 1 above)
Comment (VardaPanchal005) [reply Calculators/BMI-Calculator/BMI-Calculator.html:7]: oo okay

Comment (VardaPanchal005) [reply Calculators/BMI-Calculator/BMI-Calculator.html]: I changed it will it get updated ?

Comment (Rakesh9100) [reply Calculators/BMI-Calculator/BMI-Calculator.html]: Yes, updated

File: Calculators/BMI-Calculator/README.md (hunk 3 above)
Comment (Rakesh9100) [reply Calculators/BMI-Calculator/README.md:14]: Work on this @VardaPanchal005

Comment (VardaPanchal005) [reply Calculators/BMI-Calculator/README.md:14]: is it reflected now?

Comment (Rakesh9100) [reply Calculators/BMI-Calculator/README.md:14]: No, push your change and make commit to get it reflected @VardaPanchal005

**Comment by Rakesh9100** [comment]
Merged!! Congrats on getting your PR merged to this repo under SWOC S4 🎉 @VardaPanchal005

**Pull request merged**